install/kubernetes/helm/istio/values.yaml

```diff
@@ -283,6 +283,18 @@ global:
       enabled: false
       host: # example: metrics-service.istio-system
       port: # example: 15000
+      tlsSettings:
+        mode: DISABLE # DISABLE, SIMPLE, MUTUAL, ISTIO_MUTUAL
+        clientCertificate: # example: /etc/istio/ms/cert-chain.pem
```

Comment:
this will break kubectl edit - you cannot have trailing spaces or it will mangle everything Replace with "" instead of empty might fix it.

Reply:
Thank you. Fixed it.